### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Jamulus app](https://hosted.weblate.org/projects/jamulus/jamulus-app/).

It also includes following components:

* [Jamulus/Windows Installer](https://hosted.weblate.org/projects/jamulus/windows-installer/)

Current translation status:

Weblate translation status

### CODE REVIEW

**Review by jujudusud — APPROVED** [review]

File: src/translation/translation_it_IT.ts [hunk 1]
```diff
@@ -543,7 +543,7 @@
     <message>
         <location filename="../audiomixerboard.cpp" line="863"/>
         <source>Musician Profile</source>
-        <translation>Profilo del Musicista</translation>
+        <translation>Profilo musicista</translation>
```
Comment (pljones) [inline src/translation/translation_it_IT.ts:546]: Other labels are using "title case" in this file -- line 536, for example -- so why have these two been changed?

**Review by ann0see — APPROVED** [review]
I believe otherwise this is ok?

**Review by jujudusud — APPROVED** [review]

**Review by ann0see — APPROVED** [review]
I will merge this now.

**Comment by ann0see** [comment]
@dzpex @MarongHappy if there are any unwanted changes here, please ping me directly.

**Pull request merged**